src/components/SearchForm/SearchFormInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cx from 'classnames';
+
+type SearchFormInputProps = {
+  className?: string;
+  handleChange?: () => void;
+  searchQuery?: string;
+};
+
+export const SearchFormInput = ({
+  className,
+  handleChange,
+  searchQuery
+}: SearchFormInputProps) => {
+  const classNames = cx({
+    'search-form__input': !className,
+    [`${className}`]: className
+  });
+
+  return (
+    <input
```

Comment:
This input needs to have id="search" so label's htmlFor attribute links correctly

Reply:
This is part of something I wrote, which @DipakShrestha18 has rebased into his branch - I'll pick this up and fix

Reply:
I'll create a ticket and address in a separate PR